### ISSUE TITLE
Turmoil tags misaligned in iOS/MacOS

### ISSUE THREAD

**Issue by puuj** [terraforming-mars/terraforming-mars#4175]
I’m seeing Turmoil party symbols with random offsets on iOS webkit based browsers and Safari. This wasn’t an issue before the tags changed to symbols, afaik.
![A5316D3C-5D67-4AA4-A991-74B054BBFA8D](https://user-images.githubusercontent.com/58322/156224640-29394149-0811-47e7-aa0c-5f456ee0db06.png)

**Comment by puuj** [comment]
![17C8738C-8E0B-4BB2-BA6F-25961D933DE5](https://user-images.githubusercontent.com/58322/156225129-bc619a13-714c-40a6-bc57-a3ac55f0bdc8.jpeg)
Safari

**Comment by kberg** [comment]
Dang. Does the same thing happen with any other Mac browser, e.g.
Chrome/Firefox?

**Comment by puuj** [comment]
On iOS, yes -- all the browsers use the Webkit rendering engine. On MacOS, no - other browsers have their own rendering engine.

**Comment by kberg** [comment]
Just confirming: you have seen that it's OK on Chrome or are you guessing
from past experience?

**Comment by puuj** [comment]
Have seen both Firefox and Chrome on MacOS work correctly and have seen Firefox and Safari mess up on iOS.

**Comment by puuj** [comment]
yeah, I tried checking the developer mode information, but honestly I don't know enough CSS to understand how the absolute positions relative to the parent containers are implemented. I also don't know why it was okay before and broken now, since trawling the changes didn't make it obvious. I don't see anything online about how Webkit operates differently than other layout engines for this type of positioning, but obviously it does.

**Comment by Borbarad2** [comment]
Turmoil tags and global events were reworked completly new, and looks now different.
Is this still an issue? @K9 or @puuj 
If not this issue could be closed.
